### PR TITLE
docs:  a better chinese translation for the word 'install' in service.md

### PR DESCRIPTION
**Pull request by latteczy**
This PR is going to fix typo problem in chinese.
update service.md for zh.
/language zh

### CODE REVIEW

File: content/zh/docs/concepts/services-networking/service.md [hunk 1]
```diff
@@ -412,7 +412,7 @@ By default, kube-proxy in userspace mode chooses a backend via a round-robin alg
 任何连接到“代理端口”的请求，都会被代理到 `Service` 的backend `Pods` 中的某个上面（如 `Endpoints` 所报告的一样）。
 使用哪个 backend `Pod`，是 kube-proxy 基于 `SessionAffinity` 来确定的。
 
-最后，它安装 iptables 规则，捕获到达该 `Service` 的 `clusterIP`（是虚拟 IP）和 `Port` 的请求，并重定向到代理端口，代理端口再代理请求到 backend `Pod`。
+最后，它按照 iptables 规则，捕获到达该 `Service` 的 `clusterIP`（是虚拟 IP）和 `Port` 的请求，并重定向到代理端口，代理端口再代理请求到 backend `Pod`。
```
Comment (tengqm) [inline content/zh/docs/concepts/services-networking/service.md:415]: 安装是对的，英文原文是 'install'

Comment (latteczy) [reply content/zh/docs/concepts/services-networking/service.md:415]: 抱歉之前没跟英文版对比。这里的install翻译为`配置`是不是更符合中文习惯？

Comment (tengqm) [reply content/zh/docs/concepts/services-networking/service.md:415]: Agreed.

Comment (latteczy) [reply content/zh/docs/concepts/services-networking/service.md]: Have fixed it.  @tengqm

**Pull request merged**